### ISSUE TITLE
please support "CST_570004_WW"

### ISSUE THREAD

**Issue by rubato103** [sman591/homebridge-lg-thinq-ac#43]
The following is output to the home bridge log:

[9/20/2020, 1:36:11 PM] [LgThinqAirConditioner] Unsupported model: CST_570004_WW. Please open a GitHub issue at https://github.com/sman591/homebridge-lg-thinq-ac/issues to request support for your model. Falling back to default "POT_056905_WW" model instead.

[9/20/2020, 1:37:11 PM] [LgThinqAirConditioner] SwingModeCharacteristic: Error parsing state Error: Unsupported API value: 4
[9/20/2020, 1:37:11 PM] [LgThinqAirConditioner] SwingModeCharacteristic: Error parsing state Error: Unsupported API value: 8740
[9/20/2020, 1:37:11 PM] [LgThinqAirConditioner] SwingModeCharacteristic: Error parsing state Error: Unsupported API value: 4
[9/20/2020, 1:37:11 PM] [LgThinqAirConditioner] SwingModeCharacteristic: Error parsing state Error: Unsupported API value: 26210

It is a ceiling type system air conditioner and is controlled by temperature control, dehumidification and cooling mode. Wind swing control can be heard "beep" but is not actually manipulated.

**Comment by sman591** [comment]
Thanks for reporting! Is the swing setting the only one that fails to work properly?

**Comment by rubato103** [comment]
yes. The error message shown in the log is only one about swing mode.

**Comment by Phoendor** [comment]
Looks like it’s the sane as #23

**Comment by sman591** [comment]
Could you specify what the LG "product model" this is? If you have a link to the manual PDF that'd be great as well

**Comment by sman591** [comment]
Could you provide a copy of the snapshot for your device when this happens?

1. Turn on "debug" mode for Homebridge (start Homebridge with `-D` or in the Homebridge UI, go to "Homebridge Settings" and turn on "Homebridge Debug Mode")
2. Restart Homebridge
3. Follow the logs and wait for ~60 seconds
4. Find the line that starts with `device response.result.snapshot` and copy & paste its contents
